Packs/AttachmentsPreProcessing/Scripts/CreateFileFromPathObject/CreateFileFromPathObject.yml

```diff
@@ -0,0 +1,33 @@
+args:
+- description: path and name object passed by the "GetFilePathPreProcessing" pre-processing
+    script
+  isArray: true
+  name: object
+  required: true
+comment: This automation is being executed by the "GetFilePathPreProcessing" pre-processing
+  script that collects the paths and names of attachments of an incoming incident,
+  then passes it to this automation that reads the files and creates them in an existing
+  incident
+commonfields:
+  id: 236451eb-de96-4465-8940-4ea7f774708c
+  version: -1
+contentitemexportablefields:
```

Comment:
There are a few server fields here which should not be included in yml, try running
`demisto-sdk format -i Packs/AttachmentsPreProcessing/Scripts/GetFilePathPreProcessing/GetFilePathPreProcessing.yml`
(It should also fix your validate check)